### ISSUE TITLE
inaccessibility of options panel

### ISSUE THREAD

**Issue by rinkuyadav999** [themeum/kirki#1194]
### Issue description:
Options panel is inaccessible. For example, the enable/disable selections are not standard checkboxes, or my screen reader cannot see the checkboxes. This makes it impossible for me to select or unselect something. If these issues could be fixed, that would be great!

### Version used:
(Did you try using the develop branch from github? There's a chance your issue has already been adressed there)
Latest (TGM)

### Using theme_mods or options?
theme_mods

### Code to reproduce the issue (config + field(s))
'type'        => 'toggle', or may be with other 'input types'

**Comment by aristath** [comment]
I'm sorry for the delay here, I'll get on this as soon as possible. It might be a couple more days since I'm currently with fever.

**Comment by aristath** [comment]
I believe this was just fixed in the development branch for checkboxes, toggles & switches.
Tweaks will be included in the 2.4.0 version. 👍 

Closing this for now, if the issue persists after 2.4.0 is released (or if you test the develop branch and see that the issue still occurs there) please reopen.
